roles/icingaweb2/tasks/pgsql/users_db.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: PostgreSQL check for icingaweb admin user
+  ansible.builtin.shell: >
+    LANG=C
+    {{ _tmp_pgsqlcmd }}
+    -w -c "select name from icingaweb_user where name like '{{ _current_user.username }}'"
+  failed_when: false
+  changed_when: false
+  check_mode: false
+  register: _icingaweb2_db_user
+
+- name: Create user in Icinga Web (or reenable user / reset password)
```

Comment:
run_once is missing here.

Reply:
We should generally discuss how to handle `run_once` in database schema imports and inserts, etc. since - in theory - one could want to deploy Icinga Web on an Icinga Satellite locally.  

This would not be possible if running one play / role on 2 different hosts with their own respective database.

I'll add your suggestion for now, though.